actionpack/lib/action_controller/metal/rendering.rb

```diff
@@ -4,6 +4,17 @@ module Rendering
 
     RENDER_FORMATS_IN_PRIORITY = [:body, :text, :plain, :html]
 
+    module ClassMethods
+      # Documentation at ActionController::Renderer#render
+      delegate :render, to: :renderer
+
+      # Returns a renderer class (inherited from ActionController::Renderer)
+      # for the controller.
+      def renderer
+        @renderer ||= Renderer.for(self)
```

Comment:
There is no point.

`Renderer.for` - returns a factory for current controller. It won't matter if it's the same factory object or different one for the same controller.